source/deployment/mobile-app-deployment.rst

```diff
@@ -0,0 +1,571 @@
+- Allow users to install the application on their device.
+- Ensure a high level of security and controlled access.
+- Use existing, internal processes and tools.
+
+We recommend either of these options if you are…
+
+- Testing out the mobile applications
+- Deploying Team Edition servers using no push notifications, or push notifications from Mattermost’s `TPNS <https://docs.mattermost.com/overview/faq.html#tpns>`_ (Test Push Notification Service)
+- Deploying Enterprise Edition servers using push notifications from Mattermost’s `HPNS <https://docs.mattermost.com/mobile/mobile-hpns.html>`_ (Hosted Push Notification Service)
+
+The mobile applications provided by Mattermost work with our hosted version of the Mattermost Push Notification Service (MPNS). This represents the easier path. The Mattermost mobile applications can be deployed with or without an EMM provider. These options are explained in more detail below. 
+
+    **Note:** HPNS is compatible with EMM providers.
+
+**Option 1** - Public App Store Installation (Easiest)
+
+- Users download application via public app stores (e.g. iOS App Store or Google Play)
+- Users enter URL to your hosted Mattermost Server.
+
+**Advantages:** Very easy, mobile deployment can be done by each individual user 
+**Disadvantages:** Less control over user security, no app customization
+
+**Option 2** - Public App Store Installation with EMM Provider (Easy)
+
+- Users download application via public app stores (e.g. iOS App Store or Google Play)
+- Users enter URL to your hosted Mattermost Server.
+- Extend your organization’s security best practices/requirements via your EMM provider.
+
+**Advantages:** Easy, mobile deployment can be done by each individual user, app security and configuration can be maintained via your EMM provider 
+**Disadvantages:** No app customization
+
+***********************************************
+Distributing Custom Versions of the Mobile Apps
+***********************************************
+
+**Desired Outcomes:**
+
+- Maintain full control over the distribution of applications
+- Change the look, feel, and capabilities of the Mattermost mobile application.
+- Limit message traffic from passing through the HPNS
+
+This model is more difficult, and is recommended for organizations that can't (or don't desire) to use the HPNS. Often security and access requirements determine this, not the size of your organization.
+
+Both Google and Apple require signatures of the application and push proxy to match. This means if you build the applications, you must host your own instance of MPNS.
+
+Building the apps can be an involved process. This requires that you have the skillset to maintain and deploy packaged apps. Part of this process includes, but is not limited to :
+
+- Obtaining/providing certificates for your custom application
+- Signing your custom applications
+- Distributing your applications via public or private app stores
```

Comment:
again no such thing as public/private stores

Reply:
http://mi.extendedhelp.mobileiron.com/45/all/en/desktop/App_Catalog.htm I think this may be completed with the enterprise App Store offerings from an EMM provider. Does that still ring as incorrect? If I understand correctly there is the ability to provide app via Google Play or the App Store. There are also ways to provide Private Apps at least on the Google Play Store, but there is also a function of the EMM to allow apps to be consumed and distributed with a private/enterprise store.  Maybe we need to clarify these terms. There does seem to be evidence to establish a language around the different distribution options.

Some relevant links...

http://mi.extendedhelp.mobileiron.com/45/all/en/desktop/App_Catalog.htm
https://developer.apple.com/business/distribute/
https://ibuildapp.com/mobile-enterprise-app-store/

Reply:
resolved in meeting